s3auth-relay/pom.xml

```diff
@@ -174,6 +174,11 @@
             <groupId>com.google.guava</groupId>
             <artifactId>guava</artifactId>
         </dependency>
+        <dependency>
+            <groupId>commons-net</groupId>
+            <artifactId>commons-net</artifactId>
+            <version>3.3</version>
```

Comment:
I get a compile error both in IntelliJ and in `mvn clean install -Pqulice`. I'll remove the dependency though.